.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: python
 python:
-  - "2.6"
   - "2.7"
   - "3.4"
+  - "3.5"
```

Comment:
what about 3.6?

Reply:
in 2019, python 3.7 should be added and python 3.4 can be dropped.

Reply:
@chfw I added 3.6 and 3.7 in a separate PR #182 and 3.7 had a weird test failure - if you have a chance to look at fixing that up there, it'd be helpful.